### ISSUE TITLE
Check if you need a permit - scrambled results returned

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#748]
It's hard to understand how the addresses are listed in the results from the postcode search.
Might not be a massive issue for users but if we could make it more consistent, it would improve the user experience.

**Comment by joanneSouthern** [comment]
@EUzkuraityte any thoughts about this one ?

**Comment by EUzkuraityte** [comment]
@joanneSouthern
I'm thinking, sorting the addresses the way they are displayed by GOV.UK - listed alphabetically by street name and then sorted lowest to highest by house number, ignoring the house name.. Not saying this is ideal, just one of the options you could consider. This could definitely be one of the things you look into as part of Private Beta and user test this.
![screen shot 2017-10-30 at 15 33 27](https://user-images.githubusercontent.com/20285115/32179692-f8d87b84-bd87-11e7-8620-5a7da114bcb8.png)

**Comment by joanneSouthern** [comment]
@EUzkuraityte Hi Egle I've had a chat with Lee about ordering, the LAGAN web service orders on UPRN at the moment and we are unable to change this

**Comment by EUzkuraityte** [comment]
Makes sense... we can still look at how users react to this in user research and, if needed, keep it in the backlog for future iterations.
